README.md

```diff
@@ -180,27 +180,22 @@ const webmMediaConfig = {
   }
 };
 
-const initialMediaCapabilities = {showWarning: true};
+const initialMediaCapabilitiesInfo = { powerEfficient: true };
 
 const MyComponent = ({ videoSources }) => {
-  const { mediaCapabilities } = useMediaCapabilities(webmMediaConfig, initialMediaCapabilities);
+  const { mediaCapabilitiesInfo } = useMediaCapabilitiesDecodingInfo(webmMediaDecodingConfig, initialMediaCapabilitiesInfo);
 
   return (
     <div>
-      <video src={mediaCapabilities.supported ? videoSources.webm : videoSources.mp4} controls>...</video>
-      { mediaCapabilities.showWarning && (
-        <div class='muted'>
-          Defaulted to mp4.  
-          Couldn't test webm support,  
-          either the media capabilities api is unavailable or no media configuration was given.
-        </div>
-      ) }
+      <video src={mediaCapabilitiesInfo.supported ? videoSources.webm : videoSources.mp4} controls>...</video>
     </div>
   );
 };
 ```
 
-This hook accepts a [media configuration](https://developer.mozilla.org/en-US/docs/Web/API/MediaConfiguration) object argument and an optional `initialMediaCapabilities` object argument, which can be used to provide a `mediaCapabilities` state value when the user's browser does not support the relevant [Media Capabilities API](https://developer.mozilla.org/en-US/docs/Web/API/Media_Capabilities_API) or no media configuration was given.
+`mediaCapabilitiesInfo` value contains the three Boolean properties supported, smooth, and powerEfficient, which describe whether decoding the media described would be supported, smooth, and powerefficient.
```

Comment:
@wingleung 

Could we make a change to this naming in order to match property name?
I believe this is the last change request.
Thank you.
powerefficient => powerEfficient

Reply:
done